source/extensions/extensions_build_config.bzl

```diff
@@ -278,6 +278,12 @@ EXTENSIONS = {
     "envoy.http.original_ip_detection.custom_header":        "//source/extensions/http/original_ip_detection/custom_header:config",
     "envoy.http.original_ip_detection.xff":                  "//source/extensions/http/original_ip_detection/xff:config",
 
+    #
+    # Stateful session sticky
```

Comment:
I think we should refer to this as stateful sessions or sticky sessions, but not both.